program/databases/db_tests

```diff
@@ -6270,8 +6270,8 @@
 "006525","","b","@TOMCATADMINhtml","GET","<tt>conf\/tomcat-users\.xml<\/tt>","","","","","Default Tomcat Manager / Host Manager interface found","",""
 "006526","","b3","/getstatus","GET","License ID:","","","","","Chaos Software V-Ray status information available","",""
 "006527","","b","/platform/","GET","\.\.\/base\/index\.jsp","","","","","Platform Management Console found","",""
-"006528","http://pen-testing.sans.org/blog/pen-testing/2012/12/06/all-your-svn-are-belong-to-us","123","/.svn/entries","GET","200","","","","","Subversion Entries file may contain directory listing information.","",""
-"006529","https://www.sans.org/blog/all-your-svn-are-belong-to-us/","123","/.svn/wc.db","GET","_autoindex","","","","","Subversion SQLite DB file may contain directory listing information.","",""
+"006528","http://pen-testing.sans.org/blog/pen-testing/2012/12/06/all-your-svn-are-belong-to-us","3","/.svn/entries","GET","200","","","","","Subversion Entries file may contain directory listing information.","",""
```

Comment:
please explan this change

Reply:
I removed 1 and 2 because we have duplicates on **Interesting File** and **Information Disclosure**, I choose to keep the highest value of the risk.
1 -> is used for `Interesting File` -> Low
2 -> is used for `Misconfiguration`
3 -> is used for `Information Disclosure` -> Medium